CHANGELOG.md

```diff
@@ -12,8 +12,16 @@ This project adheres to [Semantic Versioning](http://semver.org/).
   ```rust
   use nix::fcntl::{readlink, readlinkat};
```

Comment:
This line isn't important to readers.  You can suppress it with a leading `#`, and delete the following blank line.